packages/aws-cdk/lib/api/cxapp/stacks.ts

```diff
@@ -375,3 +396,8 @@ export interface SelectedStack extends cxapi.SynthesizedStack {
    */
   originalName: string;
 }
+
+export interface Tag {
```

Comment:
@rix0rrr -- we are still separating interfaces across the cxapi right?

Reply:
@rix0rrr , any updates on this?

Reply:
Why do we need this as a public API?